subsys/bluetooth/audio/unicast_client.c

```diff
@@ -1908,6 +1911,204 @@ static int unicast_client_ase_discover(struct bt_conn *conn,
 	return bt_gatt_read(conn, &params->read);
 }
 
+static uint8_t unicast_client_pacs_avail_ctx_read_func(struct bt_conn *conn,
+						      uint8_t err,
+						      struct bt_gatt_read_params *read,
+						      const void *data,
+						      uint16_t length)
```

Comment:
Don't know whether it's GH formatting issue, but it looks like the code is not aligned here

Reply:
Was indeed missing a space :)